dotCMS/src/main/java/com/dotcms/system/event/local/type/security/CompanyKeyResetEvent.java

```diff
@@ -0,0 +1,53 @@
+package com.dotcms.system.event.local.type.security;
+
+import java.io.Serializable;
+import java.security.Key;
+
+
+/**
+ * This event is used to broadcast the info related with ResetKey event
+ */
+public class CompanyKeyResetEvent implements Serializable {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5677177)